nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -422,12 +415,91 @@ buildOperatorCodes(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
   return fbb.CreateVector(fb_op_codes);
 }
 
+/**
+ * @brief get Serialized Tensor shape
+ *
+ * @param dim Tensor dimension to convert
+ * @param type tflite Tensor type
+ * @return flatbuffers::Offset<flatbuffers::Vector<int32_t>>
+ */
+flatbuffers::Offset<flatbuffers::Vector<int32_t>>
+buildTesorShape(const TensorDim &dim, flatbuffers::FlatBufferBuilder &fbb,
+                int effective_dimension, bool is_signature = false) {
+  std::vector<int32_t> fb_dimension;
+
+  /// batch dimension is always considered with signature
+  if ((effective_dimension >> 3) & 1) {
+    fb_dimension.push_back(is_signature ? -1 : dim.batch());
+  }
+
+  for (unsigned int i = 1; i < MAXDIM; ++i) {
+    if (((effective_dimension >> (MAXDIM - i - 1)) & 1) == 0) {
+      continue;
+    }
+    fb_dimension.push_back(dim.getTensorDim(i));
+  }
+
+  return fbb.CreateVector(fb_dimension);
+}
+
+flatbuffers::Offset<flatbuffers::Vector<flatbuffers::Offset<tflite::Tensor>>>
+buildTensors(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
+  /// @todo: the actual (suqeezed) tensor dimension must be known before coming
+  /// here. For now, it is directly guessed for the fc layer
+  const auto &variables = map.getIndexMap<const Var_Grad *>().getData();
```

Comment:
I wonder that we have buildBuffers with this getData(). and Do we need it for tensor as well?

Reply:
The return of `getData()` here is indexmap of variables. This is not related to buffer :)

Reply:
right.